model/task/expected_duration.go

```diff
@@ -41,6 +39,21 @@ func getExpectedDurationsForWindow(name, project, buildvariant string, start, en
 		match[DisplayNameKey] = name
 	}
 
+	// If we pass zerotime as a starting point then the end time
```

Comment:
That seems wrong. Shouldn't it be that if we don't pass an end time, it's an unbounded window from the start time to after now?

Reply:
It does seem wrong, but its what we used to do: 

https://github.com/evergreen-ci/evergreen/commit/442b6ed9ef43004bb953ce52c2ffd950761391c3#diff-ec70289495c0b3b32aa040cc657874cfL1113